corehq/apps/domain/views.py

```diff
@@ -304,6 +304,7 @@ class BaseAdminProjectSettingsView(BaseProjectSettingsView):
         access.
     """
 
+    @method_decorator(login_and_domain_required)
     @method_decorator(domain_admin_required)
```

Comment:
is this a more general bug in the `domain_admin_required` decorator? happy to be told it's out of scope, but seems like it would be nice if it automatically ensured login/redirected and then checked permissions